frontend/public/components/monitoring.tsx

```diff
@@ -1224,16 +1200,18 @@ const RulesPage_ = (props) => (
     Header={ruleTableHeader}
     kindPlural="Alerting Rules"
     nameFilterID="alerting-rule-name"
+    labelFilter="alerts"
```

Comment:
Nit: These props were in alphabetical order (also for the other two places `MonitoringListPage` is used)

Reply:
thanks @kyoto